packages/db/src/migrations/20250425134823_granules_add_producer_granule_id.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
+  if (!await knex.schema.hasColumn('granules', 'producer_granule_id')) {
+    await knex.schema.table('granules', (table) => {
+      table
+        .string('producer_granule_id')
+        .comment('Producer Granule Id');
+    });
+
+    await knex('granules').update('producer_granule_id', knex.raw('granule_id'));
```

Comment:
Is the presumption here we're doing later work to build a reasonable migration strategy for users?   What's the execution time for L11 on a large table?

Reply:
We have the ticket to cover the migration work [CUMULUS-4069](https://bugs.earthdata.nasa.gov/browse/CUMULUS-4069)	Design and implement migration strategy